astropy/cosmology/parameters.py

```diff
@@ -168,5 +191,5 @@
 )
 
 # If new parameters are added, this list must be updated
-available = ['Planck18_arXiv_v2', 'Planck15', 'Planck13', 'WMAP9', 'WMAP7',
+available = ['Planck18', 'Planck18_arXiv_v2', 'Planck15', 'Planck13', 'WMAP9', 'WMAP7',
```

Comment:
I also wonder if we should remove the deprecated one from here.
```suggestion
available = ['Planck18', 'Planck15', 'Planck13', 'WMAP9', 'WMAP7',
```

Reply:
That would just remove it.  The cosmology objects are built off of that list, so import Planck18_arXiv_v2 would stop working.

Reply:
Oh, okay. Sorry for the noise!